wicket-core/src/test/java/org/apache/wicket/core/request/mapper/MountedMapperTest.java

```diff
@@ -714,7 +714,7 @@ public boolean isNewPageInstance()
 	@Test
 	public void placeholderEncode4()
 	{
-		PageProvider provider = new PageProvider(new MockPage())
+		PageProvider provider = new PageProvider(MockPage.class)
```

Comment:
My bad again, I meant to say: "... test follows up by changing the page **source** ..."
The line: 
      provider.setPageSource(context);
was setting up a page source that was never used because the page provider already had a page instance (the one passed as a parameter in the page provider). 

I don't know why the test sets the page source, but either way a page with no parameters will be provided making it to test the same thing. Maybe we can just remove the custom page source.